### DISCUSSION TITLE
Pedido de ajuda de um LEIGO em programação: Qual software usar pra abrir este arquivo JSON?

### DISCUSSION THREAD

**Discussion by apenasumaconta** [frontendbr/forum#2505] [Perguntas e Respostas]
Olá. Bom dia a todos.

Estou tentando traduzir um jogo +18, chamado **VillageRhapsody(乡村狂想曲)**, mas não estou conseguindo achar o arquivo que contém as LEGENDAS DAS CUTSCENES dele(colocarei 2 imagens abaixo para mostrar). Sou praticamente LEIGO em programação de computador. Apenas tenho um pouco de habilidade para localizar as palavras e frases de um arquivo de um jogo e traduzir.

Já traduzi boa parte do jogo, que está num arquivo Javascript, mas não consigo traduzir as frases das cutscenes do jogo, porque não está em nenhum arquivo Javascript do jogo(procurei em todos os arquivos javascript e não achei) e, ao tentar abrir os arquivos JSON, só abre "deformado" nas imagens abaixo.

Já tentei abrir os arquivos JSON nos programas Visual Studio Code e NotePad++ e nenhum dos dois abre. Só abre "deformado" igual a imagem abaixo.

Gostaria de saber se vocês sabem de algum programa que abre QUALQUER arquivo JSON, para ver se eu consigo localizar as legendas das cutscenes do jogo e traduzí-las. Se alguém tiver esse jogo e souber onde eu localizo essas legendas das cutscenes para eu traduzir, melhor ainda.  

As imagens dos arquivos JSON no Visual Studio Code e no Notepad++ abrem assim:

![Captura de tela 2024-09-28 152101](https://github.com/user-attachments/assets/5d4b8cf1-277d-407e-b753-59bb7f4de047)

![Captura de tela 2024-09-28 153644](https://github.com/user-attachments/assets/92a62a28-3386-48c9-8ef2-11bb112745f0)

No NotePad++, abre assim:

![Captura de tela 2024-09-30 085632](https://github.com/user-attachments/assets/d3cb98b9-3143-439a-85a2-7fb6bf147509)

Em outro forum, au usar o Notepad++, me deram a dica de de instalar o plugin JSON Viewer, clicar em gerenciador de plugins, instalar ele, ir na aba de plugins, ir em JSON  Viewer e clicar em Format JSON. Aí dá isso aqui:

![Captura de tela 2024-09-30 085717](https://github.com/user-attachments/assets/10714588-7a48-4179-a06f-744a821c1996)

![Captura de tela 2024-09-30 091346](https://github.com/user-attachments/assets/f1160c1c-99a6-4f92-9bca-574b12623964)

E em relação às cutscenes do jogo... bom, não colocarei imagem de nudez aqui. Aí vai 2 imagens de cutscenes para mostrar como exemplo. Estou tentando traduzir as LEGENDAS que vocês vêm nas imagens abaixo, mas não estou conseguindo abrir os arquivos JSON(acredito que essas frases em inglês estão lá) para traduzí-las.

![Captura de tela 2024-07-29 201538](https://github.com/user-attachments/assets/7da4f98a-8b9b-4fc7-860e-7dfbae74dcfe)

![Captura de tela 2024-07-29 201502](https://github.com/user-attachments/assets/7ab1af5f-8cad-46af-a243-ecfba6b919f9)

Agradeço desde já a ajuda de todos vocês aqui. E peço desculpas se eu tiver descumprido alguma regra deste fórum.

**Comment by jampow** [comment]
Ola @apenasumaconta. Vc tem certeza de que as legendas não fazem parte da cutscene/vídeo? Como sabe que estão neste arquivo mesmo (fiquei com essa dúvida já q vc não consegue abrir o arquivo)? pq pelo o que vi, parecem estar com algum tipo de criptografia.

Vc pode tentar outros encodings pra testar, mas não me parece ser esse o problema.

**Comment by francisco-gonzalez** [comment]
Pode ser codificação de caracteres... ja tentou um json pretty online ?
https://jsonformatter.org/json-pretty-print

**Comment by felquis** [comment]
Pelo amor no Akita on Rails, o que que está acontecendo nessa thread? 🤣
